qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -521,27 +526,27 @@ def _opt_control(property_set):
 
             # Basic steps for optimization level 2:
             # 1. RemoveIdentityEquivalent
-            # 2. Optimize1qGatesDecomposition
-            # 3. CommutativeCancellation
+            # 2. CommutativeOptimization
+            # 3. Optimize1qGatesDecomposition
             elif optimization_level == 2:
                 _opt = [
                     RemoveIdentityEquivalent(
                         approximation_degree=pass_manager_config.approximation_degree,
                         target=pass_manager_config.target,
                     ),
+                    CommutativeOptimization(),
```

Comment:
With the previous order, a long sequence of 1q gates would first be optimized, and then we run CC on the small set. Now we're running Gucci on the long sequence on 1q and then overwrite the optimizations with `Optimize1qGatesDecompostion`. Is this is a noticeable performance hit in the ASV benchmarks?